test/test_client_rpc.py

```diff
@@ -41,6 +42,11 @@ def request_cb(name, args):
     # this would have dead-locked if not async
     vim.funcs.rpcrequest(vim.channel_id, "test-event", async_=True)
     vim.run_loop(request_cb, None, None)
+
+    # This sleep is required on Travis, where it hangs with
```

Comment:
This shouldn't really be needed, but let's merge this for now. Let's mark it with `TODO`

Reply:
Done.